packages/html/ds/tailwind.config.ts

```diff
@@ -6,6 +6,11 @@ const config: Config = {
   content: ['./src/**/*.html', './src/**/*.ts'],
   theme: createTheme(),
   plugins: [],
+  safelist: [
+    // { pattern: /text-./ },
+    // { pattern: /border-./ },
+    // { pattern: /./ },
+  ],
```

Comment:
do we need this commented code ?

Reply:
I've updated it with react package version